recipes/gnutls/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+./configure --prefix="${PREFIX}" --without-libidn2 --with-included-libtasn1 --with-included-unistring --without-p11-kit
```

Comment:
Indeed it's a different version, by default gnutls wants `libidn2` which is not available on conda forge. When you set `- - without libidn2`` gnutls` then looks for `libidn11 `which is available on `conda forge `

Reply:
I'm not sure I understand the part about `p11-kit`, are you saying that this module is available by default on the environment? Because I could not find this package on `conda forge`. I think I remember that the build would fail if I did not explicitly excluded pkcs11 support but I will try again.

Reply:
No, it's not available in the default environment. I just saw that it's labeled `recommended` in Homebrew and a dependency in Debian. It appears, though, that it is only needed for encryption hardware support, so forget about it. I don't think that's relevant in most conda applications. p11-kit would have to be packaged separately.